release/github_release.py

```diff
@@ -448,7 +448,6 @@ def _sha256(version, print_=True, local=False):
         out = run(*(['shasum', '-a', '256'] + release_files(version)))
     else:
         raise ValueError('Should not get here...')
-        out = run(*(['shasum', '-a', '256', '/root/release/*']))
```

Comment:
This line could be commented out but it is better not to delete it as it shows potentially the code that needs to be there.

Reply:
I'll add it back as a comment